dpnp/dpnp_iface_mathematical.py

```diff
@@ -1759,35 +1786,70 @@ def round_(a, decimals=0, out=None):
     return around(a, decimals, out)
 
 
-def sign(x1, **kwargs):
+def sign(
```

Comment:
need to add tests in `tests/test_mathematical.py` and to verify `sign` behavior there